### PR TITLE
Add trig tabular function implementations, enable tests

### PR DESCRIPTION
**Pull request by CarlosFigueiraMSFT**
We had a test for trig tabular functions that was never enabled, because those functions were not implemented. Added the implementation for the tabular functions, fixed the test and enabled it.

### CODE REVIEW

File: src/libraries/Microsoft.PowerFx.Core/Public/Values/NumberValue.cs [hunk 1]
```diff
@@ -22,7 +22,7 @@ public override void Visit(IValueVisitor visitor)
 
         public override void ToExpression(StringBuilder sb, FormulaValueSerializerSettings settings)
         {
-            sb.Append(Value.ToString());
+            sb.Append((Value == 0) ? "0" : Value.ToString());
```
Comment (CarlosFigueiraMSFT) [inline src/libraries/Microsoft.PowerFx.Core/Public/Values/NumberValue.cs:25]: >(Value == 0) ? "0" : Value.ToString() [](http://example.com/codeflow?start=22&length=37)

We were getting a "-0" result in a few scenarios of a very small value. This fixes it.

**Comment by LucGenetier** [comment]
✅ No public API change.

**Comment by LucGenetier** [comment]
✅ No public API change.

**Comment by LucGenetier** [comment]
✅ No public API change.

**Review by Ananya-Gupta24 — APPROVED** [review]

**Comment by LucGenetier** [comment]
✅ No public API change.

**Pull request merged**